channels/cl.m3u

```diff
@@ -151,3 +151,5 @@ http://cdn.streamingmedia.cl:1935/live/vtv2/playlist.m3u8
 http://cdn.streamingmedia.cl:1935/live/vtvvina/playlist.m3u8
 #EXTINF:-1 tvg-id="ZappingMusic.cl" tvg-name="Zapping Music" tvg-country="CL" tvg-language="" tvg-logo="https://i.imgur.com/lHvoKdw.jpg" group-title="Music",Zapping (720p) [Offline]
 https://zmlive.zappingtv.com/zm_free/zm.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="KanadeTV.do" tvg-name="Kanade TV" tvg-country="LATAM" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/KdeTelevision/picture?width=300&height=300" group-title="Entertainment",Kanade TV [Not 24/7]
```

Comment:
Is it not 24/7 Anime channel? If so, it falls into Animation category

Reply:
Yeah the program seems to be fully Animation, but OP didn't know about the "Music" category, so maybe it has got some musical programming.

Reply:
If 90% of it's content is anime by design I think that doesn't matter that much

Reply:
You're right.